.github/workflows/torchserve-nightly-build.yml

```diff
@@ -6,28 +6,37 @@ on:
     - cron:  '15 11 * * *'
 jobs:
   nightly:
-    runs-on: ubuntu-18.04
+    # creates workflows for the 3 OS: ubuntu, macOS & windows
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
     steps:
-      - name: Setup Python 3.8
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-          architecture: x64
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1
+      - run: conda --version
+      - run: conda install -y conda-build anaconda-client
       - name: Checkout TorchServe
         uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          set -eux
-          pip install twine
-          sudo apt-get update
-          python ts_scripts/install_dependencies.py --cuda=cu102 --environment=dev
+          python ts_scripts/install_dependencies.py --environment=dev
```

Comment:
Because our binaries are independent of the underlying infra having CUDA or not. 
Anybody wanting to use the binaries on a CUDA enabled device has to run "python ts_scripts/install_dependencies.py --cuda=cu102 " before they install the binaries. This is a part of the torchserver README